pandas-stubs/core/frame.pyi

```diff
@@ -114,20 +114,20 @@ class _LocIndexerFrame(_LocIndexer):
         self,
         idx: IndexType
         | MaskType
-        | list[StrLike]
+        | list[HashableT]
         | tuple[
             IndexType
             | MaskType
             | slice
-            | list[StrLike]
+            | list[HashableT]
             | tuple[str | int | slice, ...],
-            list[StrLike] | slice | Series[bool] | Callable,
+            list[HashableT] | slice | Series[bool] | Callable,
         ],
     ) -> DataFrame: ...
     @overload
     def __getitem__(
         self,
-        idx: tuple[int | StrLike | tuple[StrLike, ...], StrLike],
+        idx: tuple[int | StrLike | tuple[HashableT, ...], int | StrLike],
```

Comment:
Wouldn't  a tuple of tuples return a DataFrame?

Reply:
```python
>>> df = pd.DataFrame.from_records([["a", "x", 1], ["a", "y", 2]], columns=["ab", "xy", "val"]).set_index(["ab", "xy"])
>>> df
       val
ab xy
a  x     1
   y     2
>>> df.loc[("a", "x"), "val"]
1
```
Here the argument is a tuple `(("a", "x"), "val")` and it returns a single value